### ISSUE TITLE
FR: Option to show week numbers in the leftmost column

### ISSUE THREAD

**Issue by AvijeetPrasad** [vipzhicheng/logseq-plugin-block-calendar#12]
Hi,

I was thinking that it would be great if we can also have the option to show week numbers as the left most column. For example, please see the image below.
![Screenshot 2022-07-30 at 07 50 10](https://user-images.githubusercontent.com/45669486/181876767-bc352322-1566-47d5-a7de-fbf575e94286.jpg)

**Comment by bemofan** [comment]
Yes, I would love to have this feature as well. Thank you so much

**Comment by lukas-mertens** [comment]
I just opened this github project to create exactly this issue. It would be great if this was implemented!

**Comment by senntore** [comment]
Yes! This would be a great addition to this already awesome plugin.
